autotests/testolmutility.cpp

```diff
@@ -72,7 +72,7 @@ void TestOlmUtility::verifySignedOneTimeKey()
                            aliceOlm.identityKeys().ed25519.size(), msg.data(),
                            msg.size(), sig.data(), sig.size());
 
-    QCOMPARE(std::string(olm_utility_last_error(utility)), "SUCCESS");
+    QCOMPARE(QString::fromUtf8(olm_utility_last_error(utility)), "SUCCESS"_ls);
```

Comment:
I would rather replace the whole test with the code-based one that modern Olm supports:
```suggestion
    QCOMPARE(olm_utility_last_error_code(utility), OLM_SUCCESS);
```